bookwyrm/models/book.py

```diff
@@ -363,6 +365,19 @@ def save(self, *args, **kwargs):
             for author_id in self.authors.values_list("id", flat=True):
                 cache.delete(f"author-books-{author_id}")
 
+        # Create sort title by removing articles from title
+        if self.sort_title in [None, ""]:
+            if self.sort_title in [None, ""]:
+                articles = chain(
+                    *(
+                        LANGUAGE_ARTICLES.get(language, ())
+                        for language in tuple(self.languages)
+                    )
+                )
+                self.sort_title = re.sub(
+                    f'^{" |^".join(articles)} ', "", str(self.title).lower()
+                )
```

Comment:
There are probably some edge cases were "A" is used as a noun in the title such as [A is for Activist](https://bookwyrm.social/book/257447/s/a-is-for-activist). We could probably get fancy and exclude titles that start with "A is" or even fancier with "A [verb]", but then we open a can of worms implementing custom rules for every language. Perhaps the planned addition of the ability to edit the sort title on the book edit page mitigates this concern?

Reply:
The edge cases are endless! I think letting people manually edit is the way.